nss-day-cohort-29/reactive-nutshell-copperden
# Styles

# Description
Acorn icon on login and register page

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout styles`
3. See that on both MOBILE and DESKTOP there is a cute acorn logo 

# Checklist:
